crates/autopilot/src/domain/settlement/tokenized.rs

```diff
@@ -5,7 +5,7 @@ use {
 };
 
 // Original type for input of `GPv2Settlement.settle` function.
-pub(super) type Settlement = (
+pub(super) type Solution = (
```

Comment:
`pub(super)` seems a bit sketch to me. Looks like this is only used by the `solution` file (again not sure why on-chain stuff is related to a solution).
Instead a folder structure like this could help with that.
```
/solution/tokenized
         /mod
```
Where mod can now use types that are private in `tokenized`.

Reply:
also resolved with refactor